lib/tasks/chusaku.rake

```diff
@@ -0,0 +1,8 @@
+chusaku_lib = File.expand_path(File.dirname(__FILE__, 2))
+
+desc "Add route annotations to your Rails actions"
+task chusaku: :environment do
+  require "#{chusaku_lib}/chusaku/cli"
+
+  Chusaku::CLI.new.call(ARGV[2...])
```

Comment:
You need to handle the task being invoked without any arguments:

```suggestion
  Chusaku::CLI.new.call(ARGV[2...] || [])
```

Reply:
Thank you for catching this. Confirmed that this works.